<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.util.concurrent.SettableListenableFutureTests" tests="24" skipped="0" failures="0" errors="0" timestamp="2016-11-10T07:40:14" hostname="XB-20161010YVDY" time="0.094">
  <properties/>
  <testcase name="cancelStateThrowsExceptionWhenCallingGetWithTimeout" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.022"/>
  <testcase name="cancelStateThrowsExceptionWhenCallingGet" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.0"/>
  <testcase name="setValueTriggersCallback" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.001"/>
  <testcase name="setExceptionTriggersCallback" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.0"/>
  <testcase name="setPreventsCancel" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.0"/>
  <testcase name="cancelSetsFutureToDone" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.0"/>
  <testcase name="cancelWithMayInterruptIfRunningTrueCallsOverridenMethod" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.0"/>
  <testcase name="setExceptionPreventsCancel" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.0"/>
  <testcase name="cancelDoesNotNotifyCallbacksOnSetException" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.021"/>
  <testcase name="getWaitsForCompletion" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.023"/>
  <testcase name="throwsSetExceptionWrappedInExecutionException" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.0"/>
  <testcase name="returnsSetValue" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.0"/>
  <testcase name="validateInitialValues" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.0"/>
  <testcase name="cancelPreventsExceptionFromBeingSet" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.0"/>
  <testcase name="throwsSetErrorWrappedInExecutionException" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.001"/>
  <testcase name="cancelDoesNotNotifyCallbacksOnSet" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.0"/>
  <testcase name="cancelWithMayInterruptIfRunningFalseDoesNotCallOverridenMethod" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.0"/>
  <testcase name="cancelPreventsValueFromBeingSet" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.0"/>
  <testcase name="getWithTimeoutWaitsForCompletion" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.022"/>
  <testcase name="setValueUpdatesDoneStatus" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.0"/>
  <testcase name="getWithTimeoutThrowsTimeoutException" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.002"/>
  <testcase name="setExceptionTriggersCallbackOnlyOnce" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.0"/>
  <testcase name="setValueTriggersCallbackOnlyOnce" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.001"/>
  <testcase name="nullIsAcceptedAsValueToSet" classname="org.springframework.util.concurrent.SettableListenableFutureTests" time="0.0"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
